_posts/2019-07-03-reinforce-2019-recap.md

```diff
@@ -0,0 +1,104 @@
+
+After the first day, I had already solved 4 of the 5 available daily challenges and had some issues launching the fifth challenge. Luckily with some help from the Jam lounge personnel early on the second day, I was able to get it working.
+
+I started to do the second day's challenges right after they were released and had some extra time to work on them during a workshop while waiting for Cloudformation stacks to finish creating. The challenges were surprisingly easy, so I managed to complete them all during the workshop and was the first one to finish all of them and won the first prize (Bose QC35 headphones)!
+
+![Jam lounge](/img/reinforce-2019-recap/day2-jamlounge.jpg)
+
+There was also a separate Capture the Flag event going on with a focus on more traditional security challenges (decrypting messages, binary investigation etc.) but I decided to focus on the AWS related challenges in the jam.
+
+## SDD403 - Building secure APIs in the cloud
+First workshop of the second day started at 8AM and focused on the security features available in API Gateway. It was interesting to learn more about them, as I haven't used the service that much. After creating a API Gateway from scratch, we added schema validation, VPC link, attached a WAF with some SQL injection and body size rules and Cognito authorization to it. [Link to materials](http://workshop.reinforce.awsdemo.me/)
+
+![SDD403](/img/reinforce-2019-recap/day2-sdd403.jpg)
+
+## SDD324 Setting up a DevSecOps pipeline to automate vulnerability scanning of Docker images
+A builders session where most of the one hour's time was used waiting for CloudFormation to create stacks. CoreOS Clair and Klar were used for scanning Docker images for known vulnerabilities and added as a step to a CodePipeline setup. [Link to materials](https://github.com/aws-samples/aws-codepipeline-docker-vulnerability-scan)
+
+![SDD324](/img/reinforce-2019-recap/day2-sdd324.jpg)
+
+## SEJ-001 - Security Jam
+This "hackathon" was a bit different compared to re:Invent's Game Day. This was basically similar to the security jam lounge, you had 9 different challenges that you tried to solve as a team in 4 hours. The topics in the challenges were quite varying, everything from IAM to IoT to API Gateway was used. Quite an intense session, luckily I had some talented people in my team so we managed to get 8 of the 9 challenges done and got the first place! We were also the first team to solve a sponsored challenge so we also won some Trend Micro swag and Amazon.com giftcards.
+
+![SEJ-001](/img/reinforce-2019-recap/day2-securityjam.jpg)
+
+## SDD-326 Security best practices for Amazon EKS
+Another builders session where one hour was clearly a too short time. Setting up an EKS cluster and tooling from scratch via Cloudformation took almost the whole time, so we didn't get to the actual point of the session (using IAM credentials for controlling access to pods). [Link to materials](https://github.com/meyjames/EKSKube2IAM)
```

Comment:
`where one hour was clearly a too short time` -> `where the one hour time slot was clearly not enough`